controllers/interviews.js

```diff
@@ -0,0 +1,93 @@
+const fs = require('fs');
+const { stringify } = require('querystring');
+const interviews =  require('../data/interviews.json');
+
+
+//Getting all interviews
+const getInterviews = (req, res) => {
+    return res.json(interviews);
+};
+//Getting a single interview by Job Title
+const getInterviewsByJobTitle= (req, res) => {
+    let filtInterviews= interviews.filter(item => item.job_title.includes(req.params.job_title));
+    return res.json(filtInterviews);
+};
+//Getting a single interview by Company
+
+const getInterviewsByCompany= (req, res) => {
+    let filtInterviews2= interviews.filter(item => item.company_name === req.params.company_name);
+    return res.json(filtInterviews2);
+};
+//Creating an interview
+function createInterview(req, res) {
+    const newInterview = {
+        id: (interviews.length) + 1,
+        job_title: req.query.job_title,
+        company_name: req.query.company_name,
+        date: req.query.date,
+        time: req.query.time,
+    };
+    if (!newInterview.job_title || !newInterview.company_name || !newInterview.date || !newInterview.time) {
+        return res.status(400).json({ msg: 'Please include a job title, company name, date and time' });
```

Comment:
Also, there's no need to return a message as a json format inside of an object. We could just a string instead:

```suggestion
        return res.status(400).send('Please include a job title, company name, date and time');
```